### ISSUE TITLE
Tor traffic not blocked.

### ISSUE THREAD

**Issue by Unchangeable** [besoeasy/WebTrafficCloaking#1]
I've tested your script right now and when I access the generated link from tor it doesn't redirect to blocked page link :) . Tor trafic is blocked from config. Anyway to solve this ?

**Comment by besoeasy** [comment]
yes man but this script is very old, and contains very old data for verifying the users
